src/main/java/com/example/module2/StudentMain.java

```diff
@@ -3,122 +3,152 @@
-        student.setDateOfBirth(birthDate);
-        student.setAddress(address);
-        student.setCourse(Course.SECOND);
-        student.setGroup(Group.THREE);
-        student.setFaculty(Faculty.DESIGN);
-        System.out.println(student.toString());
-
-//a) список студентов заданного факультета;
-        List<Student> studentList = new ArrayList<>();
-        studentList.add(student);
-        studentList.add(student1);
-        studentList.add(student2);
-        studentList.add(student3);
-        studentList.add(student4);
-        studentList.add(student5);
-        List<Student> filmFaculty = new ArrayList<>();
-        for (Student equalsFaculty : studentList) {
-            if (equalsFaculty.getFaculty() == Faculty.FILM) {
-                filmFaculty.add(equalsFaculty);
+        student1.setFaculty(Faculty.DESIGN);
+        System.out.println(student1);
+
+        return Arrays.asList(student1, student2, student3, student4, student5, student6);
+    }
+
+    private final static SimpleDateFormat format = new SimpleDateFormat("dd.MM.yyyy");
+
+    //a) список студентов заданного факультета;
+    private static List<Student> printFilmFacultyStudents(List<Student> studentList) {
+        List<Student> filmFacultyStudentList = new ArrayList<>();
+        for (Student student : studentList) {
+            if (student.getFaculty() == Faculty.FILM) {
+                filmFacultyStudentList.add(student);
             }
         }
-        System.out.println("Film faculty" + filmFaculty.toString());
-
-//c) список студентов, родившихся после заданного года;
-        String someDate = "01.01.1971";
-        Date someDateForEquals = format.parse(someDate);
-        List<Student> timeDate = new ArrayList<>();
-        for (Student dataEquals : studentList) {
-            if (dataEquals.getDateOfBirth().compareTo(someDateForEquals) < 0) {
-                timeDate.add(dataEquals);
+        System.out.println("Film faculty " + filmFacultyStudentList);
+        return filmFacultyStudentList;
+    }
+
+    //b) списки студентов для каждого факультета и курса;
+    private static void facultyGroupList(List<Student> studentList) {
+        List<List<Student>> globalList = new ArrayList<>();
+        for (Faculty faculty : Faculty.values()) {
+            for (Group group : Group.values()) {
+                List<Student> students = new ArrayList<>();
+                for (Student student : studentList) {
+                    if (student.getFaculty().equals(faculty) && student.getGroup().equals(group)) {
+                        students.add(student);
+                    }
+                }
+                globalList.add(students);
+            }
+        }
+        for (List<Student> list : globalList) {
+            System.out.println(list);
+        }
+    }
+
+    //c) список студентов, родившихся после заданного года;
+    private static void studentsBornEarlieThat(List<Student> studentList) throws ParseException {
```

Comment:
Action required in method name.
If you print firltered list values, add print into method name, e.g.
`private static void printStudentsBornBeforeHardcodedDate(List<Student> studentList)`